### DISCUSSION TITLE
Welcome to GrUMPy Discussions!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/GrUMPy#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bang2018** [comment]
Hello,

                Very recently I got the pythonic version to display tree of solution space. Is it possible to integrate it for large problems? Please share your thought.

Thanks and regards,
